.github/workflows/pull-request.yaml

```diff
@@ -10,7 +10,11 @@ jobs:
     steps:
     - name: Check out code
       uses: actions/checkout@v3
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
 
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
```

Comment:
It looks like these steps are unused in this workflow, or do they do something?

Reply:
Both these actions are needed for building multiple architectures without running on seperate runners that has specific architectures. 
The QEMU action setups tooling needed to emulate different architectures used by docker buildx.
And we use buildx in the `plugin.sh` build step to specify which platform should be used.

Reply:
Gotcha! Thanks 👍